app/inject/TemplateServiceProvider.java

```diff
@@ -0,0 +1,27 @@
+package inject;
+
+import com.github.jknack.handlebars.io.ClassPathTemplateLoader;
+import com.github.jknack.handlebars.io.FileTemplateLoader;
+import com.google.inject.Provider;
+import common.templates.HandlebarsTemplateService;
+import common.templates.TemplateService;
+import play.Logger;
+
+public class TemplateServiceProvider implements Provider<TemplateService> {
+    private static final String DEFAULT_TEMPLATE_PATH = "/META-INF/resources/webjars/templates";
```

Comment:
Not exactly. Handlebars needs a path where to find the templates, which in this case it is looking for them inside the sphere-sunrise-design webjar in the classpath. The webjar plugin is only useful to find the public assets of the website into the webjar.